### DISCUSSION TITLE
Goal 8: Promote sustained, inclusive and sustainable economic growth, full and productive employment and decent work for all

### DISCUSSION THREAD

**Discussion by HSCunha** [Project-Resilience/platform#30] [General]
Chat about anything related to Goal 8: [Promote sustained, inclusive and sustainable economic growth, full and productive employment and decent work for all](https://github.com/Project-Resilience/platform/blob/main/goals/goal_08.md)

**Comment by KanthiNimm** [comment]
In the wake of AI agents replacing most of the cognitive jobs, key areas of discussion for **SDG 8**

1. Defining what we call as decent work and building skills for potential work
2. What skills to develop 
3. How can we build for inclusive and sustainable economic growth 
4. What knowledge we need to acquire or tap into for productive employment and navigate going forward 
5. Finding jobs they can pivot to while integrating autonomous agents to perform their job

AI agent use cases
- Customer service - Replicant AI -  customer support AI agents

**Comment by KanthiNimm** [comment]
**Specific Research Citations:**
Acemoglu & Restrepo (2018, MIT) - Comprehensive analysis of AI's labor market effects
Brynjolfsson & McAfee's "The Second Machine Age" - Detailed technological workforce transition insights
Stanford HAI (Human-Centered AI) annual reports on AI workforce implications

**Authoritative Organizations Tracking AI Work Trends:**
OECD AI Policy Observatory
International Labour Organization (ILO) research
Brookings Institution technology workforce studies

**Comment by jbains21** [comment]
WEF - Future of Jobs Report 2025 - https://reports.weforum.org/docs/WEF_Future_of_Jobs_Report_2025.pdf

**Comment by KanthiNimm** [comment]
Sharing screenshots to dive into specifics for identifying the attributes to recommend opportunities based on the user role -

e.g.; As a data scientist looking to contribute SDG 8 work and economic growth the requirements can be displayed based on their competency. 
<img width="1499" alt="Screenshot 2025-02-14 at 11 42 23" src="https://github.com/user-attachments/assets/1658a311-6735-42bd-9e82-fb10cc2933ea" />
<img width="1495" alt="Screenshot 2025-02-14 at 11 42 38" src="https://github.com/user-attachments/assets/6c6fee8b-142a-4cfd-82e3-1bb4a6ff3806" />
<img width="1449" alt="Screenshot 2025-02-14 at 11 42 46" src="https://github.com/user-attachments/assets/76fef0a3-628c-47eb-a360-c52da6a0efde" />

We can discuss more on today's call.

**Comment by Steven-Birkinbine** [comment]
You may be interested in this public dataset on job descriptions

https://www.kaggle.com/datasets/ravindrasinghrana/job-description-dataset

Or this series of sources on how and why people change careers

https://novoresume.com/career-blog/career-change-statistics

**Comment by ngodard2** [comment]
Here is another recent resource for the public sector, published by the WEF: https://www.weforum.org/publications/matching-talent-to-the-jobs-of-tomorrow-a-guidebook-for-public-employment-services/

**Comment by ofrancon** [comment]
Google has a tool to help people change careers: https://grow.google/career-dreamer

**Comment by Steven-Birkinbine** [comment]
Google also has a tool for Interview Prep that might be worth integrating: https://grow.google/certificates/interview-warmup/
